server/controller/meetup.js

```diff
@@ -24,9 +29,14 @@ export default class meetupController {
       if (errormsg) {
         return false;
       }
-      let images = 'imageUrl';
+      let images = 'http://res.cloudinary.com/dqw7jnfgo/image/upload/v1549288478/q1pmfhfjkrgnchugokpf.jpg';
       if (req.file) {
-        images = req.file.path;
+        console.log(dataUri(req))
```

Comment:
Remove this console statement

Reply:
i apologize for this also. feedback implemented now